connector/src/main/scala/com/datastax/spark/connector/datasource/CassandraTable.scala

```diff
@@ -14,20 +14,15 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 import scala.collection.JavaConverters._
 
-case class CassandraTable(
-  session: SparkSession,
-  catalogConf: CaseInsensitiveStringMap,
-  connector: CassandraConnector,
-  catalogName: String,
-  metadata: TableMetadata,
-  optionalSchema: Option[StructType] = None) //Used for adding metadata references
+case class CassandraTable(session: SparkSession, catalogConf: CaseInsensitiveStringMap, connector: CassandraConnector, catalogName: String, metadata: RelationMetadata, optionalSchema: Option[StructType] = None) //Used for adding metadata references
 
   extends Table
     with SupportsRead
     with SupportsWrite {
 
   /* Retrieving table def from cassandra may be slow (it implies C* metadata refresh) */
   lazy val tableDef: TableDef = tableFromCassandra(connector, metadata.getKeyspace.asInternal(), name())
+  lazy val writeTableDef: TableDef = if (metadata.isInstanceOf[ViewMetadata]) tableFromCassandra(connector, metadata.getKeyspace.asInternal(), metadata.asInstanceOf[ViewMetadata].getBaseTable.asInternal()) else tableDef
```

Comment:
Could you please remove write support and throw `IllegalArgumentException` with a meaningful message when a MV write is detected? 
RDD API rejects MV writes, we would like to have the behavior consistent across all APIs.

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/15800926/207039699-fc00d8e7-5d76-41ab-8696-8f001ff8a10b.png">